web/src/app/shared/components/popup/dialog-pop-up.component.ts

```diff
@@ -32,8 +32,8 @@ export class DialogPopUpComponent implements OnInit {
   }
 
   initError() {
-    this.title = notifications.get(this.data.key).name;
-    this.message = notifications.get(this.data.key).text || this.data.passedError.error.message;
+    this.title = notifications.get(this.data.key).name || '';
+    this.message = notifications.get(this.data.key).text || this.data.passedError.error ? this.data.passedError.error.message : '';
```

Comment:
Why did you introduce the ternary operator?

Reply:
Because sometimes passedError is undefined and will cause an error, when we check passedError.error.message  just via an OR. 
These errors were not really important, but they were annoyingly filling up the console